core/modules/wbase/Task.h

```diff
@@ -75,7 +75,7 @@ struct Task {
         bool operator()(Ptr const& x, Ptr const& y);
     };
 
-    explicit Task() : _poisoned{false} {}
+    explicit Task() : entryTime(0), _poisoned{false} {}
```

Comment:
We should use consistent initialization syntax (braces vs parens) at least in the scope of single constructor.